sdk/core/azure-core/inc/azure/core/http/policies/policy.hpp

```diff
@@ -417,6 +417,35 @@ namespace Azure { namespace Core { namespace Http { namespace Policies {
           Context const& context) const override;
     };
 
+    /**
+     * @brief Helper class for creating scopes for the token request context.
+     *
+     */
+    class TokenScopes final {
+      // Remove cttor. This class is only a container for util methods.
```

Comment:
/**
     * @brief An instance of `%TokenScopes` class cannot be created.
     *
     */
    TokenScopes() = delete;

    /**
     * @brief An instance of `%TokenScopes` class cannot be destructed, because no instance
     * can be created.
     *
     */
    ~TokenScopes() = delete;